prql-compiler/src/prql.pest

```diff
@@ -101,19 +103,16 @@ range = ${ range_edge ~ ".." ~ range_edge }
 // the whole span, not just the part before `..`.
 range_edge = ${ literal? }
 
-// These separate idents from each other — i.e. `foo bar and baz` will parse into
-// [`foo bar`, `and`, `baz`], where `foo bar` is presumably a function call.
-// TODO: review operators
-operator = ${ "and" | "or" | "not" | "==" | "!=" | "=" | ">" | "<" | ">=" | "<=" | "+" | "-" | "*" | "/" | "%" }
+operator = _{ operator_unary | operator_mul | operator_add | operator_compare | operator_logical }
+operator_unary = ${ "-" }
+operator_mul = ${ "*" | "/" | "%" }
+operator_add = ${ "+" | "-" }
+operator_compare = ${ "==" | "!=" | "=" | ">" | "<" | ">=" | "<=" }
```

Comment:
I guess if we're doing these properly now, we could exclude `==`? (or close and we can assess separately if this isn't unambiguous)

```suggestion
operator_compare = ${ "!=" | "=" | ">" | "<" | ">=" | "<=" }
```